pre-compute/src/compute/pre_compute_args.rs

```diff
@@ -86,24 +86,28 @@ impl PreComputeArgs {
         // Read datasets
         let start_index = if is_dataset_required { 0 } else { 1 };
         for i in start_index..=iexec_bulk_slice_size {
+            let address = get_env_var_or_error(
+                TeeSessionEnvironmentVariable::IexecDatasetAddress(i),
+                ReplicateStatusCause::PreComputeFailedUnknownIssue, // TODO: add specific error
+            )?;
```

Comment:
As discussed, the address is the filename which should be the first extracted field.
As we don't retrieve the address if there is an error here, the error can be generic or linked to dataset ID in link.
The address can then be used in errors of following environment variables retrievals